Client/Frontend/Home/GoogleTopSiteHelper.swift

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Shared
+import UIKit
+import Storage
+
+struct GoogleTopSiteConstants {
+    // A guid is required in the case the site might become a pinned site
+    public static let googleGUID = "DefaultGoogleGUID"
+    // US and rest of the world google urls
+    public static let usUrl = "https://www.google.com/webhp?client=firefox-b-1-m&channel=ts"
+    public static let rowUrl = "https://www.google.com/webhp?client=firefox-b-m&channel=ts"
+    public static let code = [ usUrl: "firefox-b-1-m",
+                               rowUrl: "firefox-b-m"]
```

Comment:
I think this is unused.

Reply:
> I think this is unused.

Yeah this one is not used. The one in Search Telemetry class is what we use

    private static let google = ["US": "firefox-b-1-m",
                                 "ROW": "firefox-b-m"]